test/e2e/testdata/wolf-rbac/start.sh

```diff
@@ -19,6 +19,8 @@
 
 cd testdata/wolf-rbac/
 
+rm -rf db-psql.sql
```

Comment:
this is a file, we don't need `-r` argument

Reply:
Need -r, because wolf server may generate a directory with the same name, cause file download failure.